pkg/kritis/admission/admission_test.go

```diff
@@ -0,0 +1,50 @@
+/*
+Copyright 2018 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package admission
+
+import (
+	"net/http"
+	"net/http/httptest"
+	"testing"
+)
+
+func TestAdmissionReviewHandler(t *testing.T) {
+	// Create a request to pass to our handler.
+	req, err := http.NewRequest("GET", "/", nil)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	// Create a ResponseRecorder to record the response.
+	rr := httptest.NewRecorder()
+	handler := http.HandlerFunc(AdmissionReviewHandler)
+
+	handler.ServeHTTP(rr, req)
+
+	// Check the status code is what we expect.
+	if status := rr.Code; status != http.StatusOK {
+		t.Errorf("handler returned wrong status code: got %v want %v",
+			status, http.StatusOK)
+	}
+
+	// Check the response body is what we expect.
+	expected := `{"response":{"uid":"","allowed":true,"status":{"metadata":{},"status":"Success","message":"Successfully admitted."}}}`
```

Comment:
nit: this would be a bit more maintainable if you marshal and test against structs rather than a string.

Reply:
sure will do.